File: CONTROLE-REUNIAO-API/Models/Entities/Evento.cs
using System;

namespace Models.Entities
{
    public class Evento
    {
        public int EventoId { get; set; }
        public string Responsavel { get; set; }
        public DateTime Dia { get; set; }
        public TimeSpan Inicio { get; set; }
        public TimeSpan Termino { get; set; }
        public int SalaId { get; set; }
        public Sala Sala { get; set; }
    }
}

File: CONTROLE-REUNIAO-WEB/src/app/model/Evento.ts
import { Sala } from "./Sala"
import { Time } from "@angular/common"

export class Evento {

  constructor() { }

  EventoId: number
  Dia: Date
  Inicio: Time
  Termino: Time
  Responsavel: string
  
  SalaId: number
  Sala: Sala
}
